### PR TITLE
Added Wyvern Hotswap by BredWorks

### PR DESCRIPTION
**Pull request by DeskDaily**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bredworks/wyvern_hs/readme.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Bred Works Wyvern HS
+
+* Keyboard Maintainer: [DeskDaily](https://github.com/DeskDaily)
+* Hardware Supported: Atmega32u4
```
Comment (filterpaper) [inline keyboards/bredworks/wyvern_hs/readme.md:4]:
```suggestion
* Hardware Supported: Atmega32u4
* Hardware Availability: [BredWorks](https://bredworks.com/products/wyvern-hotswap-pcb)
```

A link to the vendor's website is a good way to verify that the firmware is compatible with the hardware (unless there's a good reason for omitting this bullet point from the template).

Comment (DeskDaily) [reply keyboards/bredworks/wyvern_hs/readme.md:4]: Thanks for the link! I did not know that there is already a product link for the pcb. I was only asked to make the firmware and execute the pull request

File: keyboards/bredworks/wyvern_hs/info.json
Comment (waffle87) [inline keyboards/bredworks/wyvern_hs/info.json:1]: Please run `qmk format-json path/to/info.json` on this file.

File: keyboards/bredworks/wyvern_hs/info.json [hunk 2]
```diff
@@ -0,0 +1,308 @@
+	{
+		"keyboard_name": "Wyvern HS",
+		"manufacturer": "Bred Works",
+		"url": "",
```
Comment (waffle87) [inline keyboards/bredworks/wyvern_hs/info.json:4]:
```suggestion
		"url": "https://bredworks.com/products/wyvern-hotswap-pcb",
```

**Review by dunk2k — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**